app.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'modules/game'
+require_relative 'modules/author'
+require_relative 'modules/genre'
+require_relative 'modules/music_album'
+require_relative 'modules/book_module'
+require_relative 'modules/label_module'
+require_relative 'modules/movie'
+require_relative 'modules/source'
+
+class App
+  include GameModule
+  include AuthorModule
+  include GenreModule
+  include MusicAlbumsModule
+  include BooksDataController
+  include LabelsDataController
+  include MovieModule
+  include SourceModule
+
+  def initialize
+    @options = {
+      '1' => method(:list_books),
+      '2' => method(:list_all_music_albums),
+      '3' => method(:list_all_movies),
```

Comment:
There is an error and the user is unable to see the list of all movies. Kindly make sure there is no error.
![movies_list_error](https://user-images.githubusercontent.com/35504149/176954076-a291a704-4b23-4334-9e6c-296717e04ef6.png)